samples/cpp/benchmark_app/README.md

```diff
@@ -1,66 +1,107 @@
 # Benchmark C++ Tool {#openvino_inference_engine_samples_benchmark_app_README}
 
-This topic demonstrates how to use the Benchmark C++ Tool to estimate deep learning inference performance on supported devices.
-Performance can be measured for two inference modes: latency- and throughput-oriented.
+This page demonstrates how to use the Benchmark C++ Tool to estimate deep learning inference performance on supported devices.
 
-> **NOTE**: This topic describes usage of C++ implementation of the Benchmark Tool. For the Python* implementation, refer to [Benchmark Python* Tool](../../../tools/benchmark_tool/README.md).
+> **NOTE**: This page describes usage of the C++ implementation of the Benchmark Tool. For the Python implementation, refer to the [Benchmark Python Tool](../../../tools/benchmark_tool/README.md) page. The Python version is recommended for benchmarking models that will be used in Python applications, and the C++ version is recommended for benchmarking models that will be used in C++ applications. Both tools have a similar command interface and backend.
+		
 
+## Basic Usage
+To use the C++ benchmark_app, you must first build it following the [Build the Sample Applications](../../../docs/OV_Runtime_UG/Samples_Overview.md) instructions and then set up paths and environment variables by following the [Get Ready for Running the Sample Applications](../../../docs/OV_Runtime_UG/Samples_Overview.md) instructions. Navigate to the directory where the benchmark_app C++ sample binary was built.
 
-## How It Works
+> **NOTE**: If you installed OpenVINO Runtime using PyPI or Anaconda Cloud, only the [Benchmark Python Tool](../../../tools/benchmark_tool/README.md) is available, and you should follow the usage instructions on that page instead.
 
-Upon start-up, the application reads command-line parameters and loads a network and inputs (images/binary files) to the specified device.
+The benchmarking application works with models in the OpenVINO IR (`model.xml` and `model.bin`) and ONNX (`model.onnx`) formats. Make sure to [convert your models](../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) if necessary.
```

Comment:
Why don't we talk about Paddle?
@helena-intel @kblaszczak-intel Honestly benchmark app supports all framework types which are supported in openvino. If we always will enumerate supported frameworks, we have to update a lot of documentation in case of introducing new framework support.
Supported frameworks should be enumerated on the dedicated page, I guess.

Reply:
PaddlePaddle is currently only supported for 10 selected models. We should not give the impression that our PaddlePaddle support is similar to ONNX at the moment. For now, it would be useful to add "and other supported frameworks" and link to a friendly page that explains what exactly is supported and what the known limitations are.

Reply:
The main message was that we shouldn't enumerate supported frameworks in the documentation for each tools. maybe will be better to have one common place in our documentation and references from different components to this place.

Reply:
I hope this solution is a proper common ground. It mentions PdPd and states its limitations. 
I hope we'll get to the point where we don't need to mention all of them, just state "natively supported frameworks" but at this stage, this should suffice.